src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -134,25 +147,39 @@ class WorkflowManagerActor(dataAccess: DataAccess, backend: Backend) extends Act
     }
   }
 
-  private def callStdoutStderr(workflowId: WorkflowId, callFqn: String, index: ExecutionIndex): Future[StdoutStderr] = {
+  private def isCollector(entries: Iterable[ExecutionDatabaseKey])(key: ExecutionDatabaseKey) = {
+    key.index.isEmpty &&
+      (entries exists { e =>
+        (e.fqn == key.fqn) && e.index.isDefined
+      })
+  }
+
+  private def hasLogs(entries: Iterable[ExecutionDatabaseKey])(key: ExecutionDatabaseKey) = {
+    !key.fqn.isScatter && !isCollector(entries)(key)
```

Comment:
the whole point of implicit value classes is to give you these little sorts of nuggets of sugar. This is more clear as to what the intent is - I might not care that if something `isScatter` that the underlying logic is `key.fqn.contains("$scatter")`, but i can much more quickly ascertain what the line is doing if I just see `foo.isScatter`